molecule/cookiecutter/scenario/driver/azure/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/INSTALL.rst

```diff
@@ -6,11 +6,14 @@ Requirements
 ============
 
 * An Azure credentials rc file
-* azure sdk
 
 Install
 =======
 
+Please refer to the `Virtual environment`_ documentation for installation best practices.
```

Comment:
Can we also have a mention of `--user` flag, please?

Reply:
:+1: